packages/suite/src/actions/wallet/send/sendFormEthereumActions.ts

```diff
@@ -149,6 +149,13 @@ export const composeTransaction =
             }
         } else {
             // TODO: catch error from blockbook/geth (invalid contract, not enough balance...)
+            if (estimatedFee.payload.error) return;
+            dispatch(
+                notificationsActions.addToast({
+                    type: 'metadata-unexpected-error',
+                    error: estimatedFee.payload.error,
+                }),
+            );
```

Comment:
this does not add up. if you return when `estimatedFee.payload.error` is present, you can't disaptch it on line 156. or am I missing something?

Reply:
You're right, fixed this in the new commits